src/hooks/sourceNodes/index.js

```diff
@@ -27,6 +28,17 @@ module.exports = async (
 ) => {
   const localeFallbacks = rawLocaleFallbacks || {};
 
+  if (!apiToken) {
```

Comment:
How does `apiToken` ultimately get here? Is it provided in `pluginOptions` or something?

Reply:
@sslotsky yeah, it's part of the options; the setup here is a bit different from the other plugins I've worked with, but they import this file as the `sourceNodes` API, so it has access to the Gatsby context and all that.

Reply:
Implementing the `pluginOptionsSchema` API is a better option for validating plugin options and automatically categorizing invalid options as user errors. But I believe datocms would have to be willing to bump their gatsby peer dependency version to 2.24.74.